### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by tunaytoksoz** [skiptools/skip#270] [Q&A]
During the build phase, I am encountering an error. What could be the problem?

<img width="734" alt="Screenshot 2024-10-22 at 10 33 33" src="https://github.com/user-attachments/assets/9885dd07-d55b-4c26-9822-a7455a66f2fe">

**Comment by marcprux** [comment]
You can look at the log file that the is referenced at the end of the build, but I think it may be that the app/module name has a number in it, which I believe may not be allowed. Can you try again without the number and let us know if it works?

**Reply by marcprux** [reply]
Actually, the number should be fine. Can you look through the log file (`/tmp/skip-init-2024-10-2207:32:35Z.txt`) to see if you can spot the error message? Feel free to post the excerpt here for further assistance.
